database/migrations/2025_01_22_190453_create_user_season_score_aggregates_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('user_season_score_aggregates', function (Blueprint $table) {
+            $table->bigInteger('user_id')->unsigned();
+            $table->integer('season_id')->unsigned();
+            $table->float('total_score');
```

Comment:
this explodes on values as small as 7M

```
SQLSTATE[22003]: Numeric value out of range: 1264 Out of range value for column 'total_score' at row 1 (Connection: mysql, SQL: insert into `user_season_score_aggregates` (`user_id`, `season_id`, `total_score`) values (189617, 40, 7643867))
```

It needs to be either actual float or double (`float` here is `double(8,2)`). Or maybe rounded to int instead. Maybe @peppy can weigh in.

Reply:
is that a laravel thing which maps `float` to `double(8,2)`?

i'm fine with any suitable data type. i don't think this table should get big enough for the choice to matter.

Reply:
yeah afaict up to laravel 10 it's not possible to use this syntax to create actual float column

Reply:
went ahead and just changed it to regular double